samples/src/Microsoft.Azure.IIoT.App/src/Pages/Browser.razor

```diff
@@ -308,7 +308,7 @@
         {
             if (node.Id == samples.NodeId)
             {
-                node.Value = samples.Value.ToObject<string>();
```

Comment:
are these changes in razor related to PR title?

Reply:
Good catch - this was a bug fix - it was excepting for me if value is null